docs/ref/signals.txt

```diff
@@ -405,6 +405,13 @@ Arguments sent with this signal:
 ``using``
     The alias of database on which a command will operate.
 
+``plan``
+    .. versionadded:: 1.8
+
+        The migration plan; a list of two elements tuples containing
```

Comment:
deindent one level (so it's at the same level as `.. versionadded:: 1.8`) [and below]